api/errors.go

```diff
@@ -34,11 +33,15 @@ func WriteError(w http.ResponseWriter, err error) {
 
 	// Write errors in the response writer
 	if rl, ok := w.(logging.ResponseLogger); ok {
+		logErr := err
+		if u, ok := err.(*acme.Error); ok {
+			logErr = u.Err
+		}
```

Comment:
You might want to create a method like this in the acme package and use just that one.